walker.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""A Random Walk Simulation For Multiple Walkers"""
+
+# Python code for 2D random walk.
+# Source: https://www.geeksforgeeks.org/random-walk-implementation-python/
+import sys
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def create_walkers(number_of_steps, number_of_walkers):
+    """
+    Creates a list of numpy arrays which each hold the coordinates of a walker. The coordinates
+    are randomly chosen in a range from 0 to the number of steps
+    :param number_of_steps: the number of steps the walker will make. This defines the
+    dimension of the array which represents the walker
+    :param number_of_walkers: the number of walkers defines the number of arrays, which
+    will be created
+    :return: a list, the 'scene' which contains all walkers, so all arrays containing coordinates
+    """
+    scene = []
+    for _ in range(number_of_walkers):
```

Comment:
Is this ` _ ` a convention for not used variables in loops? Haven't seen it before!

Reply:
Yes! Didn't knew it before as well, but after pylint was reporting an unused counter I search for solutions and found that this is the convention. As soon as I used it pylint didn't criticised it anymore ;)